src/App.tsx

```diff
@@ -43,6 +46,20 @@ function getWsUri(id: string) {
   );
 }
 
+function useKeyboardCtrlIntercept(key: string, reaction: (evt: KeyboardEvent) => unknown) {
```

Comment:
Nit, but `evt` -> `event` would be clearer. Also, it's more common to write `(event: KeyboardEvent) => void`, which is actually a special case alias for `unknown` when in the return type of a function in TypeScript.

Reply:
Sure on `event`. `void` instead of `unknown` is also fine if you want. The reason I usually stick with unknown is that that accepts more methods (not just functions that return nothing, but also functions that return anything at all), and it still expresses that you cant work with the return value IN THIS PLACE.

Because of that I can reuse the same function in a few places, where otherwise I'd have to create a meaningless wrapper that ignores the return value, or another version.

In this case its not relevant, but I just made it a habit to do this for these reasons.

But its your repository, if you want void there you'll have it.